packages/backoff/backoff.0.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Exponential backoff mechanism for OCaml"
+maintainer: ["Carine Morel <carine@tarides.com>"]
+authors: ["KC Sivaramakrishnan <sk826@cl.cam.ac.uk>"]
```

Comment:
```suggestion
authors: ["KC Sivaramakrishnan <kc@tarides.com>"]
```

I suspect this is sadly long out of date :-) /cc @kayceesrk 

Submitter of PR is different from author/maintainers; should you also be in there somewhere @polytypic?

Reply:
I'm in the LICENSE.md.

For background, there are several exponential backoff implementations in various multicore libraries.  I redesigned the backoff implementation in Kcas to reduce overheads and to allow functions to be efficiently parameterized with configured backoffs while working on Kcas and we are now factoring it out as a separate library (work done by @lyrm) so that it can be conveniently used in other places (e.g. Kcas, Saturn (in general), implementation of scheduler friendly locks, ... any place where someone wants to write code using atomics).  Perhaps at some point we could have backoff in the Stdlib.